modules/seqexec/web/client/src/main/resources/less/style.less

```diff
@@ -429,32 +432,81 @@ body {
   margin-bottom: 1em;
 }
 
-.SeqexecStyles-acProgressRow {
+.SeqexecStyles-tableDetailRow {
   display: flex;
   flex-grow: 1;
   flex-direction: column;
   width: 100%;
   height: 100%;
-  padding: @table_row_left_padding;
+  padding: @table_detail_row_top_padding @table_detail_row_right_padding @table_detail_row_bottom_padding @table_row_left_padding;
   z-index: 100;
   position: relative;
   margin-left: 20px;
 }
 
+.SeqexecStyles-nodAndShuffleDetailRow {
+  display: flex;
+  justifyContent: spaceBetween;
+}
+
+.SeqexecStyles-nodAndShuffleControls {
+  margin-left: 14px;
+}
+
 .SeqexecStyles-observationProgressBar {
   margin: 0 !important;
+  height: 1em !important;
   width: 100%;
 }
 
 .SeqexecStyles-observationBar {
-  height: 1em !important;
+  height: 100% !important;
 }
 
 .SeqexecStyles-observationLabel {
   text-align: center;
   width: 100%;
 }
 
+.SeqexecStyles-dividedProgress {
+  width: 100%;
+  display: flex;
+}
+
+.SeqexecStyles-dividedProgressSectionLeft {
+  border-top-right-radius: 0 !important;
```

Comment:
Did you try without the `!important`? I can imagine it is required but it is good to avoid it if possible

Reply:
Yes, I always try first without the `!important`.